src/internal/minikubetestenv/deploy.go

```diff
@@ -396,8 +396,16 @@ func waitForLoki(t testing.TB, lokiHost string, lokiPort int) {
 }
 
 func waitForPgbouncer(t testing.TB, ctx context.Context, kubeClient *kube.Clientset, namespace string) {
+	waitForLabeledPod(t, ctx, kubeClient, namespace, "app=pg-bouncer")
+}
+
+func waitForPostgres(t testing.TB, ctx context.Context, kubeClient *kube.Clientset, namespace string) {
+	waitForLabeledPod(t, ctx, kubeClient, namespace, "app.kubernetes.io/name=postgresql")
+}
+
+func waitForLabeledPod(t testing.TB, ctx context.Context, kubeClient *kube.Clientset, namespace string, label string) {
```

Comment:
btw, it would be nice if this t.Log'd every retry; which label selector it's waiting on, and the status of the matches (pod phase if != running, which container is not ready, etc.).

Reply:
Edit: it looks like all this info is already logged in require.noErrorWithinTRetry, so I will refrain from doing it a second time.